client/driver/rkt.go

```diff
@@ -257,9 +282,13 @@ func (d *RktDriver) Validate(config map[string]interface{}) error {
 	fd := &fields.FieldData{
 		Raw: config,
 		Schema: map[string]*fields.FieldSchema{
+			// image is only required if pod_manifest was not passed in
```

Comment:
Please make all your comments full sentences as per golang preferred guide, unless the project style guides does not recommend that?